scipy/stats/tests/test_distributions.py

```diff
@@ -5221,7 +5225,12 @@ def test_cdf_against_r(self, r_case_result):
     def test_moment_vectorization(self):
         # Test moment broadcasting. Calls `_munp` directly because
         # `rv_continuous.moment` is broken at time of writing. See gh-12192
-        m = stats.studentized_range._munp([1, 2], [4, 5], [10, 11])
+
+        # Silence invalid value encountered warnings. Actual problems will be
+        # caught by the result comparison.
+        with np.errstate(invalid='ignore'):
+            m = stats.studentized_range._munp([1, 2], [4, 5], [10, 11])
```

Comment:
This is good, but I just got a new one in `test_compare_r`.
https://dev.azure.com/scipy-org/SciPy/_build/results?buildId=16112&view=logs&j=e3555eda-5afc-5d78-d2e9-cf9756f98375&t=220e105d-2d02-5073-9f3b-414490a3ddd5&l=1298

These are only happening in 32-bit builds, I think, so they don't worry me very much. We know that the integrand is very challenging numerically, and we did the best we could. I'm OK with warnings in the tests as long as the results are good.

Reply:
Just added an ignore to test_against_R.